exporter/otlphttpexporter/otlp.go

```diff
@@ -124,7 +125,9 @@ func (e *exporter) pushLogs(ctx context.Context, ld plog.Logs) error {
 }
 
 func (e *exporter) export(ctx context.Context, url string, request []byte) error {
-	e.logger.Debug("Preparing to make HTTP request", zap.String("url", url))
+	var span = trace.SpanFromContext(ctx)
+	var traceId = span.SpanContext().TraceID().String()
+	e.logger.Debug("Preparing to make HTTP request", zap.String("url", url), zap.String("traceId", traceId))
```

Comment:
What happens if we are exporting a metric?  Would it print out an empty traceId?  Can we only include the traceId if we have one?

Reply:
traceID is the traceID of the request in the collector (not from the telemetry we transport), we always have one if the otel-trace-go is configured.